CHANGES.md

```diff
@@ -21,6 +21,7 @@ Release Notes.
 * Add the plugin for mssql-jtds 1.x.
 * Add the plugin for mssql-jdbc 6.x -> 9.x.
 * Fix the default ignore mechanism isn't accurate enough bug.
+* Add the plugin for spring-kafka 1.x.
```

Comment:
I think you indicate 1.3.x only? At least tested? Do I miss anything?

Reply:
> I think you indicate 1.3.x only? At least tested? Do I miss anything?

Yes. You are right, we have only test 1.3.x. Changed.